Source/WebCore/platform/graphics/Color.h

```diff
@@ -209,9 +209,9 @@ class Color {
         unsigned blue;
         unsigned alpha;
     };
-    DebugRGBA debugRGBA() const;
+    WEBCORE_EXPORT DebugRGBA debugRGBA() const;
```

Comment:
Do you need to export these?

Reply:
That's fair.

Reply:
Moved to https://github.com/WebKit/WebKit/pull/46767